packages/lab/src/progress/CircularProgress/Circle.tsx

```diff
@@ -0,0 +1,22 @@
+import { CSSProperties } from "react";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import circularProgressCSS from "./CircularProgress.css";
```

Comment:
do you need these styles here?

Reply:
actually don't think we need Circle.tsx at all, going to delete